### PR TITLE
Add Google Analytics tracking code

### LINKED ISSUE #5: Set up Google Analytics in config.yml
A GA account was created recently (see #4). We now need to add it to `_config.yml`.

### PR DESCRIPTION
**Pull request by marlonbernardes**
Add google analytics tracking code to `_config.yml`. 

Fix #5

(WIP: I still need to test it)

### CODE REVIEW

**Comment by marlonbernardes** [comment]
It didn't work as expected. It seems that I need to remove the `{% raw %}` tag in order to output a `config.yml` property.

**Comment by marlonbernardes** [comment]
Yes, that was it.

with `{% raw %}`  the properties `site.google_universal_analytics` and  `site.google_universal_analytics_cookiedomain` were not being replaced by their respective values:

```
ga('create', '{{ site.google_universal_analytics }}', '{{ site.google_universal_analytics_cookiedomain }}');
```

After I removed the `{% raw %}` tag, the result was correct:

```
ga('create', 'UA-60941725-1', 'auto');
```

File: _includes/google_analytics.html [hunk 1]
```diff
@@ -1,13 +1,11 @@
 {% if site.google_universal_analytics %}
-  {% raw %}
```
Comment (marlonbernardes) [inline _includes/google_analytics.html:2]: I had to remove this tag, is this the best approach to fix the issue I mentioned in the comments? It seems good to me.

**Comment by rafaeleyng** [comment]
@marlonbernardes I think you did the right thing. The `{% raw %}` tag is a Liquid tag to disable tag processing. Both `{% %}` and `{{ }}` won't be processed. [Docs](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers)

**Pull request merged**